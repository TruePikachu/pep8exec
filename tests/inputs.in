foo bar baz qux
12
16
