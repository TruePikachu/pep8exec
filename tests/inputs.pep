	; Test file for reading input
	STRO	str0,d
	LDX	0,i
	LDA	0,i
rdLoop:	CHARI	inBuf,x
	LDBYTEA	inBuf,x
	ADDX	1,i
	CPX	256,i
	BREQ	tooLong
	CPA	'\x0D',i
	BREQ	numLoop
	CPA	'\x0A',i
	BRNE	rdLoop

numLoop:STRO	str2,d
	STX	numChar,d
	DECO	numChar,d
	STRO	str3,d
	DECI	numChar,d
	LDA	numChar,d
	STX	numChar,d
	CPA	numChar,d
	BRNE	wrong
	STRO	str5,d
	SUBX	1,i
	BR	outLoop
wrong:	STRO	str4,d
	BR	numLoop

outLoop:SUBX	1,i
	CHARO	inBuf,x
	BRNE	outLoop
	CHARO	'\n',i
	STOP

tooLong:STRO	str1,d
	STOP

str0:	.ASCII	"Enter text, terminate with newline: \x00"
str1:	.ASCII	"Ummm...that is too long...\x00"
str2:	.ASCII	"You entered \x00"
str3:	.ASCII	" characters...how many? \x00"
str4:	.ASCII	"IN"
str5:	.ASCII	"CORRECT!\n\x00"
numChar:.WORD	0
inBuf:	.BLOCK	255
	.BLOCK	1
	.END
