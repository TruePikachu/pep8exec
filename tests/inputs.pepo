41 00 5F C8 00 00 C0 00 00 4D 00 D3 D5 00 D3 78
00 01 B8 01 00 0A 00 5B B0 00 0D 0A 00 24 B0 00
0A 0C 00 09 41 00 9F E9 00 D1 39 00 D1 41 00 AC
31 00 D1 C1 00 D1 E9 00 D1 B1 00 D1 0C 00 48 41
00 C7 88 00 01 04 00 4E 41 00 C5 04 00 24 88 00
01 55 00 D3 0C 00 4E 50 00 0A 00 41 00 84 00 45
6E 74 65 72 20 74 65 78 74 2C 20 74 65 72 6D 69
6E 61 74 65 20 77 69 74 68 20 6E 65 77 6C 69 6E
65 3A 20 00 55 6D 6D 6D 2E 2E 2E 74 68 61 74 20
69 73 20 74 6F 6F 20 6C 6F 6E 67 2E 2E 2E 00 59
6F 75 20 65 6E 74 65 72 65 64 20 00 20 63 68 61
72 61 63 74 65 72 73 2E 2E 2E 68 6F 77 20 6D 61
6E 79 3F 20 00 49 4E 43 4F 52 52 45 43 54 21 0A
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
00 00 00 zz
