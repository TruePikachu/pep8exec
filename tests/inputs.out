Enter text, terminate with newline: You entered 16 characters...how many? INCORRECT!
You entered 16 characters...how many? CORRECT!
xuq zab rab oof
